inspec/a2-iam-legacy-integration/controls/migrated_custom_access.rb

```diff
@@ -0,0 +1,97 @@
+# encoding: utf-8
+# copyright: 2018, Chef Software, Inc.
+# license: All rights reserved
+
+# This test suite must ONLY run on iam_v1_force_upgrade_to_v2.sh.
+# That test scenario tests a specific case that upgrades an old version of 
+# Automate using IAM v1 to the latest Automate, which force-upgrades it to IAM v2
+# The test a2-iam-v1-integration/controls/custom_access_to_migrate.rb adds a custom policy to that older version.
+# In this test we verify that the policy migrated correctly as part of the force-upgrade.
+
+require_relative '../../constants'
+
+title 'v1 custom policy migrates to v2'
+
+control 'iam-migrated-custom-legacy-policies-1' do
```

Comment:
this is a new test that tests custom policy migration that depends on `custom_access_to_migrate.rb` to run.